src/docker/Dockerfile

```diff
@@ -131,22 +131,17 @@ RUN wget -O /usr/local/bin/docker-compose --progress=dot:giga "https://github.co
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
 # Install Podman. Get available versions by running the command `dnf list podman`
-ARG PODMAN_VERSION="3.2.3"
+ARG PODMAN_VERSION="3.3.1"
 ENV PODMAN_VERSION=${PODMAN_VERSION}
 RUN dnf -y install podman-${PODMAN_VERSION} \
   && dnf clean all \
   && rm -rf /var/cache/yum/ \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
 # Install Skopeo. Get available versions by running the command `dnf list skopeo`
-ARG SKOPEO_VERSION="1.3.1"
+ARG SKOPEO_VERSION="1.4.2"
 ENV SKOPEO_VERSION=${SKOPEO_VERSION}
 
-# A seperate binary version needs to be set because, for some unknown reason, skopeo's package verion and binary version are different.
-# This is caused by Red Hat's packaging and there isn't much we can do about it.
-ARG SKOPEO_BINARY_VERSION="1.3.2"
-ENV SKOPEO_BINARY_VERSION=${SKOPEO_BINARY_VERSION}
-
```

Comment:
Did Red Hat fix the package version and binary version differences?

Reply:
yeah, as far as I can tell. I tested the unit tests beforehand